.github/CONTRIBUTOR_GUIDE/ct/AppName.md

```diff
@@ -0,0 +1,231 @@
+# **AppName<span></span>.sh Scripts**
+ `AppName.sh` scripts found in the `/ct` directory. These scripts are responsible for the installation of the desired Application. For this guide we take `/ct/snipeit.sh` as example.
+
+
+## 1. **File Header**
+
+### 1.1 **Shebang**
+- Use `#!/usr/bin/env bash` as the shebang.
+
+```bash
+#!/usr/bin/env bash
+```
+### 1.2 **Import Functions**
+- Import the build.func File.
+- When developing your own Script, change the link to your own repository.
+
+> [!CAUTION]
+> Before opening a Pull Request, change the link to point to the community-scripts repo.
+
+Example for development:
+```bash
+source <(curl -s https://raw.githubusercontent.com/[USER]/[REPO]/refs/heads/[BRANCH]/misc/build.func)
+```
+
+Example for final Script: 
+```bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+```
+
+### 1.3 **Metadata**
+- Add clear comments for script metadata, including author, copyright, and license information.
+
+Example:
+```bash
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: [YourUserName]
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: [SOURCE_URL]
+```
+
+> [!NOTE]: 
+> - Add your username and source URL
+> - For existing scripts, add "| Co-Author [YourUserName]" after the current author
+
+---
+
+## 2 **Variables and Function import**
+> [!NOTE]
+> You need to have all this set in your Script, otherwise it will not work!
+
+### 2.1 **Default Values**
+- This sections sets the Default Values for the Container.
+- `APP` needs to be set to the Application name and must represent the filenames of your scripts.
+- `var_tags`: You can set Tags for the CT wich show up in the Proxmox UI. Don´t overdo it! 
+
+>[!NOTE]
+>Description for all Default Values
+>| Variable | Description | Notes |
+>|----------|-------------|-------|
+>| `APP` | Application name | Must match ct\AppName.sh |
+>| `TAGS` | Proxmox display tags without Spaces, only ; | Limit the number |  
```

Comment:
var_tags or TAGS?

Reply:
TAGS

Reply:
I assume the question is here because the table says `TAGS` and a few lines above it says `var_tags`. This confuses me as well. Or are those two different context?